src/Core/Users/login.class.php

```diff
@@ -0,0 +1,79 @@
+<?php
+require_once "UserInterface.php";
+
+Class login implements UserInterface{

```

Comment:
Class implementing a interface, `UserInterface` is generally named minus `interface`, ie `User`.

Class name needs to be follow the output of `ucfirst`. 

Don't really need to have `.class` in its file name.